molo/surveys/tests/test_rules.py

```diff
@@ -57,13 +57,14 @@ def setUp(self):
         self.number = PersonalisableSurveyFormField.objects.create(
             field_type='number', label='Number Field', page=self.survey)
 
-        # Create survey submission
+    # Create survey submission
         data = {
             self.singleline_text.clean_name: 'super random text',
             self.checkboxes.clean_name: ['choice 3', 'choice 1'],
             self.checkbox.clean_name: True,
             self.number.clean_name: 5
         }
+
```

Comment:
@miltontony  you added these. Should I remove them or keep them?

Reply:
You can remove it